### DISCUSSION TITLE
Setting Up a Token

### DISCUSSION THREAD

**Discussion by NicholeCaprio** [community/community#113203] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a token created, but I need to know how to use it. I don't know where I can place the code.

**Comment by SebastienXia** [comment]
## Hello, @NicholeCaprio 👋

> I've found this Github Documentation for you about the token in Github click [here](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)

I don't know if this will help you :)

**Comment by vineeth-au** [comment]
With limited info here, I can try to answer the best way I can. This is a sample Python code which would use github token to create a PR programmatically.

```
from github import Github


  def create_pr(self):
      github = Github(login_or_token=<YOUR_GITHUB_TOEN_GOES_HERE>)
      try:
          return (
              github.get_organization("some-org")
              .get_repo("some-repo-name")
              .get_releases()
              .get_page(0)
              .__getitem__(0)
              .body
          )
      except Exception as error:
          print("Unable to find repo some-repo-name. You might have to update your GitHub access key", error) raise error
```

Hope this helps...

**Comment by codal-tshah** [comment]
Can you provide more specific details about tokens, where it is generated from ?
